Catering WebAPI/Entities/Order.cs

```diff
@@ -10,7 +10,7 @@ public class Order
         public string Status { get; set; }
         [Column(TypeName = "decimal(18,2)")]
         public decimal Prize { get; set; }
-        public CateringType CateringTypes { get; set; }
+        public List<CateringType> CateringTypes { get; set; }
```

Comment:
Shouldnt be like that. I recoomend create another table called fe OrderPosition with OrderId, DietType, CaloriesVariant, Price